service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpSend.java

```diff
@@ -112,6 +112,14 @@ public void send(String deviceId, Message message) throws IOException
             this.reactor = Proton.reactor(this);
             // Codes_SRS_SERVICE_SDK_JAVA_AMQPSEND_12_008: [The function shall start the Proton reactor object]
             this.reactor.run();
+            try
```

Comment:
this.reactor.run(); is already on line #114. Why is it triggered again on #117 ?